lib/io/reader.toit

```diff
@@ -302,6 +302,14 @@ abstract class Reader implements old-reader.Reader:
   Returns -1 if $throw-if-absent is false and the $byte is not in the remaining data.
   */
   index-of byte/int --to/int?=null --throw-if-absent/bool=false -> int:
+    absent := :
```

Comment:
Maybe we should then just provide a method?
For example?
```
index-of byte/int --to/int?=null [--if-absent] -> int?:
```

But probably for a different PR.